test/unit_inbroker_parallel_SUITE.erl

```diff
@@ -61,6 +61,9 @@ groups() ->
               login_of_passwordless_user,
               set_tags_for_passwordless_user
             ]},
+          {access_control, [parallel], [
+              state_update
```

Comment:
We already discussed not to add this test, right?

Reply:
Yes, will remove it.